scripts/upload-image.sh

```diff
@@ -1,5 +1,3 @@
-set -e
```

Comment:
don't we want to fail on error?

Reply:
Probably yes but it was already there in the original script. @rynoV what do you think?

Reply:
I can't remember if there was a valid reason when I've deleted, let me try to re-add it.